doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -552,6 +553,11 @@ Bluetooth libraries and services
   * Updated by deleting reset in progress flag from settings storage instead of storing it as ``false`` on factory reset operation.
     This is done to ensure that no Fast Pair data is left in the settings storage after the factory reset.
 
+* :ref:`bt_le_adv_prov_readme` library:
+
+  * Changed the :kconfig:option:`CONFIG_CONFIG_BT_ADV_PROV_FAST_PAIR_ADV_BUF_SIZE` allowed range and increased its default value to 20.
+    This is done to align the buffer size to new, 1 byte larger, Fast Pair not discoverable advertising data size.
```

Comment:
```suggestion
  * Changed the allowed range of the :kconfig:option:`CONFIG_CONFIG_BT_ADV_PROV_FAST_PAIR_ADV_BUF_SIZE` Kconfig option and set its default value to 20.
    This is done to increase the buffer size to 1 byte more than the Fast Pair not discoverable advertising data size.
```

Reply:
ah, I understood it wrongly. I have updated the comment. I think that we need not mention "I byte more", since user could make out from the new value set for the Kconfig option.